nf_core/pipeline-template/nextflow.config

```diff
@@ -266,11 +259,48 @@ manifest {
     homePage        = 'https://github.com/{{ name }}'
     description     = """{{ description }}"""
     mainScript      = 'main.nf'
-    nextflowVersion = '!>=23.04.0'
+    nextflowVersion = '!>=23.10.0'
     version         = '{{ version }}'
     doi             = ''
 }
 
+{% if nf_schema -%}
+// Nextflow plugins
+plugins {
+    id 'nf-schema@2.1.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+}
+
+validation {
+    defaultIgnoreParams = ["genomes", "helpFull", "showHidden", "help-full", "show-hidden"] // The last 4 parameters are here because of a bug in nf-schema. This will be fixed in a later version
+    help {
+        enabled = true
+        command = "nextflow run $manifest.name -profile <docker/singularity/.../institute> --input samplesheet.csv --outdir <OUTDIR>"
+        {%- if is_nfcore %}
+        beforeText = """
+-\033[2m----------------------------------------------------\033[0m-
+                                        \033[0;32m,--.\033[0;30m/\033[0;32m,-.\033[0m
+\033[0;34m        ___     __   __   __   ___     \033[0;32m/,-._.--~\'\033[0m
+\033[0;34m  |\\ | |__  __ /  ` /  \\ |__) |__         \033[0;33m}  {\033[0m
+\033[0;34m  | \\| |       \\__, \\__/ |  \\ |___     \033[0;32m\\`-._,-`-,\033[0m
+                                        \033[0;32m`._,._,\'\033[0m
+\033[0;35m  ${manifest.name} ${manifest.version}\033[0m
+-\033[2m----------------------------------------------------\033[0m-
+"""
+        afterText = """${manifest.doi ? "* The pipeline\n" : ""}${manifest.doi.tokenize(",").collect { "  https://doi.org/${it.trim().replace('https://doi.org/','')}"}.join("\n")}${manifest.doi ? "\n" : ""}
+* The nf-core framework 
+  https://doi.org/10.1038/s41587-020-0439-x
+
+* Software dependencies
+  https://github.com/${manifest.name}/blob/master/CITATIONS.md
+"""{% endif %}
```

Comment:
nope you can't access subworkflow functions in a config

Reply:
does it need to be in the config then. can't we load this somewhere else?

Reply:
not as far as I know. the only way to get these in the new help messages is via a config because they are now printed before the pipeline actually executes